include/onnxruntime/core/framework/sparse_tensor.h

```diff
@@ -3,12 +3,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+#if !defined(ORT_MINIMAL_BUILD)
+
 #include "core/framework/data_types.h"
 #include "core/framework/tensor_shape.h"
 #include "core/framework/tensor.h"
 
-using namespace onnxruntime::common;
-
```

Comment:
I guess this is the reason of a bunch of build breaks :smile:

Reply:
Yeah - one bad header had a lot of impact over time.
Thanks for fixing the build breaks!